src/bidiMapper/modules/context/NavigationTracker.ts

```diff
@@ -283,13 +301,10 @@ export class NavigationTracker {
   frameNavigated(url: string, loaderId: string, unreachableUrl?: string) {
     this.#logger?.(LogType.debug, `frameNavigated ${url}`);
 
-    if (
-      unreachableUrl !== undefined &&
-      !this.#loaderIdToNavigationsMap.has(loaderId)
-    ) {
-      // The navigation failed before started. Get or create pending navigation and fail
-      // it.
+    if (unreachableUrl !== undefined) {
```

Comment:
Nit: 
```suggestion
    if (unreachableUrl) {
```

Reply:
I don't think if correctly covers edge case of `unreachableUrl =""`

Reply:
ACK